confidential-data-hub/hub/src/hub.rs

```diff
@@ -86,12 +97,64 @@ impl DataHub for Hub {
                 || async move { initialize_image_client(self.config.image.clone()).await },
             )
             .await?;
-        let manifest_digest = client
+
+        let image_info = client
             .lock()
             .await
             .pull_image(image_url, Path::new(bundle_path), &None, &None)
             .await?;
-        Ok(manifest_digest)
+
+        #[cfg(feature = "ttrpc")]
```

Comment:
Remind me does the AA not have grpc support? Should we be giving people some kind of warning if the right feature is not enabled?